osquery/tables/system/windows/authenticode.cpp

```diff
@@ -125,6 +126,49 @@ void generateRow(Row& row, const SignatureInformation& signature_info) {
   }
 }
 
+bool getCatalogPathForFilePath(const std::wstring path,
```

Comment:
I think this should be passed by const reference too.

Reply:
Sure, updated. Compiler will optimize these cases. Modern compilers have made me lazy. :)